modal.js

```diff
@@ -0,0 +1,18 @@
+
+var openModalbtn = document.querySelector("#open-modal")
+
+var modal = document.getElementById("myModal");
+
+var span = document.querySelector("#close-modal");
```

Comment:
Try using ES6+ syntax instead of var.

Reply:
Although this doesn't give error, but you can try renaming span to something more specific, like 'closeModalBtn'.

Reply:
The UI looks awesome, but this might be an oversight but there are no code snippets visible in the deploy preview.

Reply:
sure, will change that, thanks!